sources/fakeni/main.py

```diff
@@ -30,16 +33,28 @@
 counter = 0
 
 try:
+    READ_PERIOD = int(1 / SAMPLE_RATE * 1000000000) # Up to nanosecond accuracy
+    relative_last_read_time = time.time_ns()
+    print(READ_PERIOD / 1000000)
     while True:
         start = time.time()
         # send a tuple of when the data was recorded and an array of the data for each channel
+        relative_timestamps = list(range(
+                relative_last_read_time,
+                relative_last_read_time + READ_PERIOD * READ_BULK,
+                READ_PERIOD,
+            ))
+            
         data = {
             "timestamp": start,
-            "data": {f"Fake{i}": [random.random() for _ in range(READ_BULK)] for i in range(CHANNELS)}
+            "data": {f"Fake{i}": [random.random() for _ in range(READ_BULK)] for i in range(CHANNELS)},
+            "relative_timestamps_nanoseconds": relative_timestamps,
```

Comment:
We don't need to generate and send the range, we can just send the starting timestamp and sample rate right?